pymc3/gp/cov.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, input_dim, active_dims=None):
             self.active_dims = np.arange(input_dim)
         else:
             self.active_dims = np.array(active_dims)
-            assert len(active_dims) == input_dim
+            assert len(active_dims) == input_dim, "One boolean value per dimension required"
```

Comment:
best practice to actually raise an error instead of using the assert keyword -- could you change this to 

    if len(active_dims) != input_dim:
        raise AssertionError(message)

(or perhaps a better type of error)